tests/test_envs.py

```diff
@@ -28,9 +28,10 @@ def task_tester(task_type):
     # Make sure sim is stopped
     og.sim.stop()
 
-    # Make sure GPU dynamics are enabled (GPU dynamics needed for cloth)
+    # Make sure GPU dynamics are enabled (GPU dynamics needed for cloth) and no flatcache
     gm.ENABLE_OBJECT_STATES = True
     gm.USE_GPU_DYNAMICS = True
+    gm.ENABLE_FLATCACHE = False
```

Comment:
why no flatcache?

Reply:
We get an assertionerror, because cloth isn't compatible with flatcache

Reply:
Got it!